packages/notifier-seeder/src/notifierErrors.ts

```diff
@@ -0,0 +1,16 @@
+import { InternalError } from "pagopa-interop-models";
+
+const notifierErrorCode = {
+  eventV1ConversionError: "9999",
+} as const;
+
+export type NotifierErrorCode = keyof typeof notifierErrorCode;
```

Comment:
Since internal errors are not converted to API responses, the code mapping to a number is not strictly needed, so you could have a string code literal without the object that maps it to a code - see for example the errors in file manager/queue manager

Reply:
make sense